File: LLM-Translations/GPT-4/codenet/Java/C/s059916837.c
#include <stdio.h>
#include <string.h>
#include <limits.h>

int find(const char *a, const char *b, int len);

int main() {
    char S[1001], T[1001];
    scanf("%s", S);
    scanf("%s", T);
    int min = INT_MAX;
    for(int i = 0; i <= strlen(S) - strlen(T); i++){
        min = (min < find(S + i, T, strlen(T))) ? min : find(S + i, T, strlen(T));
    }
    if(min != INT_MAX){
        printf("%d\n", min);
    } else {
        printf("%lu\n", strlen(T));
    }
    return 0;
}

int find(const char *a, const char *b, int len) {
    int calc = 0;
    for(int x = 0; x < len; x++) {
        if(a[x] != b[x]) {
            calc++;
        }
    }
    return calc;
}

/*  */

File: LLM-Translations/GPT-4/codenet/Java/Go/s059916837.go
package main

import (
	"fmt"
	"math"
	"os"
	"strings"
)

func main() {
	var S, T string
	fmt.Scan(&S, &T)

	min := math.MaxInt32
	for i := 0; i <= len(S)-len(T); i++ {
		min = int(math.Min(float64(min), float64(find(S[i:i+len(T)], T))))
	}

	if min != math.MaxInt32 {
		fmt.Println(min)
	} else {
		fmt.Println(len(T))
	}
}

func find(a, b string) int {
	calc := 0
	for x := 0; x < len(a); x++ {
		if a[x] != b[x] {
			calc++
		}
	}
	return calc
}

// 
